### DISCUSSION TITLE
Papers

### DISCUSSION THREAD

**Discussion by dfrepos** [youxch/Inverse-design-of-patch-antennas#6] [General]
Hi Dr. You,

I recently came across your rep and found the topic extremely relevant and interesting to my current research. I am especially intrigued by the inverse design aspect as i've done something similar recently, but in photonics. I am currently working on the design of microwave antenna for non destructive testing, and I believe that having access to the full text of your paper would be of great help. Could you kindly share a PDF copy of your papers? My current institution does not have access to the repositories you've published them in.  Thank you for considering my request. 

Best regards,
Dr. Davi Franco Rêgo
Professor EBTT  
e-mail: davifr@ifba.edu.br 
lattes: http://lattes.cnpq.br/9226556976460182
Departamento de Eletrotécnica | Department of Electrotechnics
Instituto Federal da Bahia | Federal Institute of Bahia

**Comment by youxch** [comment]
Hi Dr. Davi Franco Rêgo,

Thank you for your interest in my research. I’m glad to hear that you found the topic relevant to your current work. I have sent a PDF copy of my paper to you. I hope it will be helpful for your research on microwave antenna design for non-destructive testing.
[Energy_Efficient_Design_of_Low-Profile_Wideband_Microstrip_Patch_Antennas_Using_Deep_Learning.pdf](https://github.com/youxch/Inverse-design-of-patch-antennas/files/15070287/Energy_Efficient_Design_of_Low-Profile_Wideband_Microstrip_Patch_Antennas_Using_Deep_Learning.pdf)
[Micro   Optical Tech Letters - 2024 - You - Inverse design of reflective metasurface antennas using deep learning from.pdf](https://github.com/youxch/Inverse-design-of-patch-antennas/files/15070292/Micro.Optical.Tech.Letters.-.2024.-.You.-.Inverse.design.of.reflective.metasurface.antennas.using.deep.learning.from.pdf)

Best regards, 
Xi Chong You
Master’s Student at ShanghaiTech University
